### ISSUE TITLE
Missing files?

### ISSUE THREAD

**Issue by NotNeoAU** [ILoveAppleJuice/pet-simulator99-autofish#1]
Seems theres a few missing files such as shrek.png and t2.png.
Managed to work out that shreks just the background so thats fine but I'm unsure about t2.png.
I managed to get it working about 15% of the time using a image of the fishing minigame bobber.

If you have discord I'd love to talk, my users nxeo

**Comment by ILoveAppleJuice** [comment]
I didnt expect anyone to actually see this lol my discord is iloveapplejuicee

**Comment by ILoveAppleJuice** [comment]
You're right though, the shrek is the background and t2 is an image of the bobber.
